native/android/.idea/misc.xml

```diff
@@ -4,7 +4,7 @@
   <component name="FrameworkDetectionExcludesConfiguration">
     <file type="web" url="file://$PROJECT_DIR$" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_13" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Is it intended that language level is 13, but project-jdk-name is still 8.

And why have you updated this?

Reply:
This should either be JDK 8 or JDK 15. JDK 15 is the current version of java whereas jdk8 is the legacy version

Reply:
will revert. IntelliJ updated it by itself, as I did have another jdk on my system